DAWdleUnitTesting/UnitTesting.cpp

```diff
@@ -811,4 +811,64 @@ namespace MemoryTests {
 		char buffer2[10] = "hello";
 		EXPECT_EQ(memcmp(buffer1, buffer2, 3), 0);
 	}
-}
\ No newline at end of file
+}
+
+namespace ByteBuffer {
+
+	const StrA str = "dawdleDotcom"sa;
+
+	TEST(ByteBuff, Wrap) {
+		ByteBuf test;
+		U8 testValue = 'a';
+		U8* testdata = &testValue;
+		U32 dataSize = sizeof(testdata);
+		test.wrap(testdata, dataSize);
+		EXPECT_EQ(test.bytes, 'a');
```

Comment:
Since test.bytes is a pointer this will check if the address of the byte buffer is equal to the numerical ascii code for a. You need some more custom code to validate that the contents of the buffer is correct

Reply:
It's worth noting that the wrap method literally makes a wrapper around the pointer you pass in. In other words, test.bytes == testdata, so that's what you'd want to check if you're testing the wrap method.